controllers/Users.js

```diff
@@ -88,7 +88,7 @@ function UsersController() {
     if (req.user.admin) {
       Performance.findNextOrOpenWindow()
       .then((performance) => {
-        res.render('users/admin', { user: req.user, performance: Performance.format(performance) });
+        res.render('users/admin', { user: req.user, performance: performance && performance.toJSON() });
```

Comment:
You'll have to test this, but I believe `JSON.stringify()` will call `toJSON()` if it is available, so you might be able to just pass `performance` without having to call `toJSON()` explicitly.